cfme/tests/automate/test_file_import.py

```diff
@@ -31,3 +37,56 @@ def test_domain_import_file(import_datastore, import_data):
             3. Import should work. Check imported or not.
     """
     assert import_datastore.exists
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(automates=[1720611])
+@pytest.mark.parametrize("files", ["datastore_blank.zip", "dialog_blank.yml"],
```

Comment:
sorry for nitpicking but why files ? as I see you pass files one by one. so, it is definitely "file". and since you are testing upload. I'd call it upload_file or blank_file or test_file.